requirements/edx/github.txt

```diff
@@ -28,6 +28,8 @@ git+https://github.com/mfogel/django-settings-context-processor.git@b758c3930862
 git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 # The officially released version of django-debug-toolbar-mongo doesn't support DJDT 1.x. This commit does.
 git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c4aee6e76b9abe61cc808
+# custom opaque-key implementations for ccx
+-e git+https://github.com/jazkarta/ccx-keys.git@d1c6b2e71315b6c6a45be89912e895546d422765#egg=ccx-keys
```

Comment:
Needs an update since the latest changes on https://github.com/jazkarta/ccx-keys/pull/2, but e6b0370 looks good.